src/connectedk8s/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+1.10.0
+++++++
+* Added support for arc gateway feature.
```

Comment:
```suggestion
* Add support for arc gateway feature.
```

Reply:
it is better to specify which commands/parameters are impacted

Reply:
done, thanks for reviewing!